servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -1844,7 +1844,6 @@ void RendererCanvasRenderRD::occluder_polygon_set_shape(RID p_occluder, const Ve
 	ERR_FAIL_COND(!oc);
 
 	Vector<Vector2> lines;
-
```

Comment:
You should avoid changing whitespace in lines that don't relate to your PR as they create a lot of noise in the git history.

Reply:
I think this was an accident in removing some code for debug output. Just to keep this out of history, I should just squash the commits, right?